scripts.py

```diff
@@ -1018,3 +1017,11 @@ class OPDSForDistributorsReaperScript(OPDSImportScript):
     IMPORTER_CLASS = OPDSForDistributorsImporter
     MONITOR_CLASS = OPDSForDistributorsReaperMonitor
     PROTOCOL = OPDSForDistributorsImporter.NAME
+
+class LaneResetScript(LibraryInputScript):
+    """Reset a library's lanes based on language configuration or estimates
+    of the library's current collection."""
+
+    def process_library(self, library):
+        create_default_lanes(self._db, library)
+        self._db.commit()
```

Comment:
This is dangerous enough that it should take a special argument and act on a 5-second delay, like `adobe_account_id_reset`.

Reply:
And maybe also have a dry run option so you can see what the new lanes will be.